google-cloud-bigtable/lib/google/cloud/bigtable/row_filter/chain_filter.rb

```diff
@@ -489,7 +488,7 @@ def column_range range
             add(RowFilter.column_range(range))
           end
 
-          # Get number of filter count in chain filters
+          # Gets number of filter count in chain filters.
```

Comment:
The grammar in this sentence is still a bit off.

Reply:
Is it accurate to say "Gets the number of filters in a chain."?

Reply:
I think it should be "Gets the number of filters in the chain." or "Gets the number of filters in the chain filter."